src/vs/workbench/contrib/notebook/common/notebookCommon.ts

```diff
@@ -927,6 +927,7 @@ export const NotebookSetting = {
 	focusIndicator: 'notebook.cellFocusIndicator',
 	insertToolbarLocation: 'notebook.insertToolbarLocation',
 	globalToolbar: 'notebook.globalToolbar',
+	stickyScroll: 'notebook.stickyScroll',
```

Comment:
We may want to follow `editor.stickyScroll.enabled` in case we want to add more options to the sticky scroll feature.